servo/connectors/kubernetes.py

```diff
@@ -1796,173 +2237,28 @@ def adjust(self, adjustment: Adjustment, control: Control = Control()) -> None:
             resource_name = "memory" if name == "mem" else name
             requirements = getattr(self.container_config, resource_name).requirements
             self.container.set_resource_requirements(
-                name, value, requirements, clear_others=True
+                resource_name, value, requirements, clear_others=True
```

Comment:
@blakewatters This bug seems to be present in master as well FYI: https://github.com/opsani/servox/blob/b4fe0f84ea069a541698dfcb8e6deaf170580da5/servo/connectors/kubernetes.py#L1796-L1800

It was causing a 422 during my regression testing:
```
2020-11-10 14:42:50.854 | INFO     | servo.connectors.kubernetes:rollback:2202 | kubernetes[adjust] - adjustment failed: rolling back controller... ((422)
Reason: Unprocessable Entity
HTTP response headers: <CIMultiDictProxy('Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Date': 'Tue, 10 Nov 2020 20:42:50 GMT', 'Content-Length': '1469')>
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"Deployment.apps \"web-main\" is invalid: [spec.template.spec.containers[0].resources.limits[mem]: Invalid value: \"mem\": must be a standard resource type or fully qualified, spec.template.spec.containers[0].resources.limits[mem]: Invalid value: \"mem\": must be a standard resource for containers, spec.template.spec.containers[0].resources.requests[mem]: Invalid value: \"mem\": must be a standard resource type or fully qualified, spec.template.spec.containers[0].resources.requests[mem]: Invalid value: \"mem\": must be a standard resource for containers]","reason":"Invalid","details":{"name":"web-main","group":"apps","kind":"Deployment","causes":[{"reason":"FieldValueInvalid","message":"Invalid value: \"mem\": must be a standard resource type or fully qualified","field":"spec.template.spec.containers[0].resources.limits[mem]"},{"reason":"FieldValueInvalid","message":"Invalid value: \"mem\": must be a standard resource for containers","field":"spec.template.spec.containers[0].resources.limits[mem]"},{"reason":"FieldValueInvalid","message":"Invalid value: \"mem\": must be a standard resource type or fully qualified","field":"spec.template.spec.containers[0].resources.requests[mem]"},{"reason":"FieldValueInvalid","message":"Invalid value: \"mem\": must be a standard resource for containers","field":"spec.template.spec.containers[0].resources.requests[mem]"}]},"code":422}
)
```